### DISCUSSION TITLE
Help about using Codeberg translation

### DISCUSSION THREAD

**Discussion by xavizardKnight** [Helium314/SCEE#672] [Q&A] [answered]
Heyas! Yesterday I discovered this app while browsing through the OSM wiki, and I love it. The extra personalization, the traffic restrictions overlay, and the options for custom missions and custom overlays are super useful (even though I still need to learn the syntax). Great job!

I want to help translate SCEE into the Catalan language, but I don't really understand how to use the Codeberg interface (sorry if this is a silly simple question); I created the Catalan language there, but it's empty of strings.
Do I need to download the English XML, translate it locally and upload it into the Catalan section? What do I do now to start working with the translation?

**Comment by mcliquid** [comment, marked as answer]
Hi, if you already have an account on Codeberg, you should be able to easily translate Catalan [here](https://translate.codeberg.org/translate/scee/app/ca/?q=state:%3Ctranslated). It works for me.

**Reply by xavizardKnight** [reply]
Thank you! I tried looking around yesterday, but I couldn't find anything, it showed that there were no strings to translate. I'll start working now on translating.

**Reply by mnalis** [reply]
> I tried looking around yesterday, but I couldn't find anything, it showed that there were no strings to translate

Yeah I can confirm that, Catalan was claiming that there were 0 strings to translate few hours after this discussion was opened when I tried it too... Perhaps translate.codeberg.org needs some time for it to sync new language after it is added? :man_shrugging: 

(there were same issues with Danish translation too before, and it "just worked" later, might've been related: https://github.com/Helium314/SCEE/discussions/468)
